### DISCUSSION TITLE
Isotropy in the MeerKAT Survey of the cosmic radio dipole

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#267] [Cosmology Models]
"The MeerKAT Absorption Line Survey Data Release 2: Wideband continuum catalogues and a measurement of the cosmic radio dipole"
[https://arxiv.org/abs/2408.16619](https://arxiv.org/abs/2408.16619)

The assumption of isotropy in the number counts of radio sources seems to work on Gpc scales?

**Comment by LifeIsStrange** [comment]
> The final results are consistent with the CMB dipole in terms of direction and amplitude, unlike many recent measurements of the cosmic radio dipole made with other centimetre wavelength catalogues,

I have only read the abstract but it seems to say that their finding is consistent with the CMB dipole, meaning that the number count is NOT isotropic and over and underdensities follow the famous dipole shape.

> The declination effect seen in the MALSdata dominates the dipole signal and introduces an artificial dipole effect which points towards the south pole with high amplitude.

They say after that they correct this systematic, the "declination effect" and yet they still find a dipole signal. I don't understand the consequence of correcting for their systematic.

**Reply by RedshiftDrift** [reply]
The 'dipole shape' is not considered an anisotropy but a result of our motion relative to the "CMB rest-frame".

The systematic effect biases the data away from the result they want.  After correcting for that systematic effect, the smaller dipole signal appears with the same amplitude as measured with the CMB.  Since the CMB is isotropic (***after removing contributions from multipoles***) their conclusion is that the assumption of isotropy is correct.

I'm not saying I agree with their logic...

**Comment by sahil5d** [comment]
Figures 11 and 12 don't seem to corroborate their "recovering the CMB dipole".

The direction of the dipole is not that close

<img width="441" alt="fig11" src="https://github.com/user-attachments/assets/54a96f88-5eb6-44b0-95a3-0839696e81bb">

The amplitude of the dipole is not that close either

<img width="429" alt="fig12" src="https://github.com/user-attachments/assets/588f844d-74f7-4538-86de-fd3ea3009437">

The Secrest et al. (2022) paper is the best of the lot so far in my opinion.
